Toolset/home.livecodescript

```diff
@@ -1207,6 +1208,28 @@ command revInternal__InitialiseLibraries
    try
       revIDEDeployInitialize
    end try
+   
+   -- Load libraries in the Ext externals bundle
+   if there is a folder revEnvironmentExtPath() then
+      set the folder to revEnvironmentExtPath()
+      repeat for each line tExternal in the folders
+         if tExternal is empty or tExternal is "." or tExternal is ".." then
+            next repeat
+         end if
+         set the folder to revEnvironmentExtPath() & "/" & tExternal
+         
+         local tFiles
+         put the files into tFiles
+         filter tFiles with "*lib*.livecode*"
+         
+         repeat for each line tFile in tFiles
+            if there is a stack tFile then
+               start using stack tFile
+            end if
```

Comment:
@montegoulding Just thinking through how the external libraries work here.... Rather than issuing a "start using", would it be better to do:

`send "revLoadLibrary" to stack tFile`

so that the script can choose to insert itself into the back, set itself as a behavior, or whatever it wants to do?